changelog/dmd.catch-statement-nothrow.dd

```diff
@@ -0,0 +1,46 @@
+The presence of catch statements no longer raises an error with `-betterC` or `-nothrow`
+
+They are now simply ignored, as the try body can be assumed to never throw.
+This makes it easier to write common code that works both with and without Exceptions enabled, for example:
+
+---
+void log(O)(O o)
+{
+    try
+    {
+        print("log: ", o.toString())
+    }
+    catch (Exception e)
+    {
+        print("log: ", e.msg);
+        throw e;
+    }
+}
+---
+
+To make this compatible with `-betterC`, you had to write something like this:
+
+---
+void log(O)(O o)
+{
+    version (D_BetterC)
```

Comment:
When giving an example of old code which shouldn't be used anymore, please use `D_Exceptions` at least.

Reply:
Done